dokkaJavaApi/src/main/kotlin/dokka.java.api.JavaPlugin.kt

```diff
@@ -34,9 +34,19 @@ abstract class JavaDokkaPlugin : DokkaPlugin() {
                 override fun invoke(sourceSet: DokkaConfiguration.DokkaSourceSet, context: DokkaContext): DModule =
                     createSourceToDocumentableTranslator(context, SourceSetWrapper(sourceSet))
             }
-        } override dokkaBase.descriptorToDocumentableTranslator
+        }  override dokkaBase.psiToDocumentableTranslator
     }
 
+    // Just turn off another translator sicnce multiple overrids does not work
+    val diableOtherTranslator by extending {
+        CoreExtensions.sourceToDocumentableTranslator providing { ctx ->
+            object : SourceToDocumentableTranslator {
+                override fun invoke(sourceSet: DokkaConfiguration.DokkaSourceSet, context: DokkaContext): DModule =
+                    TODO()
+            }
+        }  override dokkaBase.descriptorToDocumentableTranslator applyIf{ false }
+    }
+    
```

Comment:
I just want to say there that dokka has special interface for plugins called: `WithUnsafeExtensionSuppression`. Maybe we should use it. I think list representation of suppresed extensions is more transparent

Reply:
Hmm, that may be a good idea. I will add a follow up PR for that